### ISSUE TITLE
In power point, Nvda does not read current slide (only repeats the first slide) in the  slide show view if nvda is running in Nepali locale.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4701]
**Reported by dhankuta on 2014-12-16 02:25**
If NVDA is set to Nepali locale; In slide show view of power point; either moving to next or previous slides or pressing the down/up arrow key for paragraph reading, , , instead of reading the contents of the current slide,  nvda repeately reads the first slide contents. Instantly, if the locale is switched to other language from general setting, it reads normally. Again if the locale is set to Nepali, the same problem of repeatetion of the last normally read slide begins instead to read the current slide contents. No matter what is the language of the contents of the slide. Nepali is written in Devanagari and in many cases texts written in Devanagari is taken as Hindi texts by default.

**Comment by ehollig** [comment]
Closing, as it sounds like this issue has been resolved from https://github.com/nvaccess/nvda/issues/4701#issuecomment-155330834
